modules/installation-osp-kuryr-octavia-configuration.adoc

```diff
@@ -240,3 +240,99 @@ If Kuryr uses `ovn` instead of Amphora, it offers the following benefits:
 * Reduced network latency.
 * Increased service creation speed by using OpenFlow rules instead of a VM for each Service.
 * Distributed load balancing actions across all nodes instead of centralized on Amphora VMs.
+
+
+[id="installation-osp-kuryr-octavia-driver_upgrade_{context}"]
+== Upgrading Octavia provider drivers after installation
+
+You can upgrade a cluster that is deployed on {rh-openstack-first} from the Octavia Amphora
```

Comment:
Not sure how this renders, but to me it is not clear. Perhaps better to be explicit:

An OCP cluster that was deployed in OSP 13 and then upgraded to OSP16 can be updated to use Octavia OVN provider driver instead of the Octavia Amphora driver


Note that what is upgraded is the OSP cluster, and we just allow the installed OCP cluster to take advantage of the new OSP 16 features (mainly related to octavia)

Reply:
@luis5tb Sure. Is this information only ever going to be relevant for people who upgrade from OSP 13 to 16, or are there other paths, too?

Reply:
No, this information is needed by the OCP admin to know that now, if OSP supports ovn-octavia (as in the OSP13 to OSP16 upgrade), OCP cluster can take advantage of it by triggering the described steps.

Reply:
It may also be that for some reason ovn-octavia was disabled in OSP16 and then the OSP admin enables it. In that case the OCP admin may also want to take advantage of it and trigger the steps

Reply:
@luis5tb And, come to think of it, does it make sense to refer to this process as "upgrading" or "updating" something on OCP, or just making a configuration change to allow for the OVN driver?

Reply:
yeah, to me both "switching|updating from amphora to ovn-octavia driver to back up OCP services" seems ok. It is not that you upgrade the OCP cluster, but that you adjust it to take advantage of OSP features